code/src/index.css

```diff
@@ -1,13 +1,273 @@
+*{
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
 body {
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  list-style: none;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+main{
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  min-height: 100vh;
+  background-image : url('./assets/bg.jpg');
+}
+
+.main-container h1{
+  color: white;
+  margin: 20px;
+}
+
+.main-container{
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  margin: 10px;
+  position: absolute;
+}
+
+.main-container button{
+  border-radius: 20px;
+  width: 100px;
+  height: 40px;
+  background-color: #fd9a89;
+  font-weight: bold;
+  color: white;
+  border: 0 none;
+  cursor: pointer;
+  padding: 10px 5px;
+  margin: 10px 5px;
+}
+
+.main-container button:hover{
+  background-color: transparent;
+    border: 2px solid #fd9a89;
+    cursor: pointer;
+}
+
+.App {
+  display: flex;
+  align-items: center;
+  position: relative;
+}
+
+body {
```

Comment:
Perhaps the body and .App could come earliner in the code, together with other general page styling